distribution/test/itests/test-itests-common/src/test/java/ddf/common/test/PostTestConstruct.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+package ddf.common.test;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Pax Exam OSGi containers cannot execute {@code @PostConstruct} in the container.
+ *
+ * {@code @PostTestConstruct} will execute after object creation before each test run when
+ * combined with {@link PaxExamRule}. This will provide a suitable location for post-construction
+ * modifications, after {@code @Inject} annotations have been run and before the test is initialized.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.METHOD)
+public @interface PostTestConstruct {
```

Comment:
Just curious. Did we try using javax.annotation.PostContruct instead of creating your own and see if Karak/PaxExam calls the annotated method at the right time?